internal/metrics/metrics.go

```diff
@@ -11,6 +11,20 @@ var StartTimeGauge = promauto.NewGauge(prometheus.GaugeOpts{
 	Help: "The timestamp of the time when the app started",
 })
 
+// ********************* GRAPH METRICS ******************
+
+// GraphAssetsGauge reports the number of nodes in the graph
+var GraphAssetsGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+	Name: "go_graphkb_graph_assets_gauge",
```

Comment:
opt: I like to add `total` to these so its very clear the is a total count of something

Reply:
I was wondering about that as well and thought that we can compute it in prometheus. Do you still want me to add it anyway? I'm fine with both.

Reply:
go_graphkb_graph_assets_total_gauge   done